### ISSUE TITLE
Callback for handle_errors

### ISSUE THREAD

**Issue by rafaeliga** [wyeworks/boom#62]
Hello,

I have an app that we handle json errors, using handle_errors:

```
def handle_errors(conn, assigns) do
  handle_errors(get_format(conn), conn, assigns)
end

defp handle_errors("json", %{status: status_code} = conn, %{reason: reason}) do
  conn
  |> json(%{error: %{status: status_code, message: Exception.message(reason)}})
  |> halt()
end

defp handle_errors(_format, conn, _assigns), do: conn
```

Is there a option to continue using this with Boom?

What do you think of a setting/config that sets a callback at the [end of Boom handle_errors](https://github.com/wyeworks/boom/blob/master/lib/boom_notifier.ex#L87)?

**Comment by iaguirre88** [comment]
@rafaeliga We just merged a PR that implements this feature. It will be released in the next version 🙌 
Thanks for your suggestion ❤️

**Comment by iaguirre88** [comment]
@rafaeliga 

Released in v0.7.0
https://hex.pm/packages/boom_notifier/0.7.0

**Comment by rafaeliga** [comment]
> @rafaeliga
> 
> Released in v0.7.0 https://hex.pm/packages/boom_notifier/0.7.0

Thats awesome @iaguirre88 ! Thank you 💪

I was missing some documentation when I tried the PR on master, gonna try another look now.

**Comment by iaguirre88** [comment]
@rafaeliga 

Take a look at this section. If you think it's not clear enough feel free to ask or improve the readme
https://github.com/wyeworks/boom#setup-when-handle_errors2-is-already-being-used
